packages/syft/src/syft/core/node/common/node.py

```diff
@@ -104,6 +107,9 @@ def __init__(
         signing_key: Optional[SigningKey] = None,
         verify_key: Optional[VerifyKey] = None,
         db_path: Optional[str] = None,
+        TableBase: Any = None,
```

Comment:
I am not quite ahppy with this, because the node should be able to pick it's storage in a more generic way. We should split functionality here 100%.

Reply:
Tudor and I chatted about this offline - SQLAlchemy is generic enough but this interface should be cleaned up with a wrapper class around TableBase, and the other SQLAlchemy specific pieces here.